### PR TITLE
Allow watching multiple files at once

### PR DESCRIPTION
**Pull request by Kobzol**
Fixes: https://github.com/spirali/nelsie/issues/62

### CODE REVIEW

File: python/nelsie/__main__.py [hunk 1]
```diff
@@ -1,40 +1,73 @@
...
+    # Re-execute the file
+    importlib.machinery.SourceFileLoader("slides", filenames[0]).load_module()
+
     s = watch._WATCH_SET
     watch._WATCH_SET = None
-    s.add(filename)
-    return list(s)
+
+    to_watch = list(filenames)
+    to_watch.extend(list(s))
```
Comment (spirali) [inline python/nelsie/__main__.py:51]: This is not correct. Watched images will grow forever with duplicates everytime the slides are rebuild.

**Review by spirali — CHANGES_REQUESTED** [review]

File: python/nelsie/__main__.py (hunk 1 above)
Comment (Kobzol) [reply python/nelsie/__main__.py:51]: The second call should have just passed `source_filenames`, not `files`, thanks.

Comment (spirali) [reply python/nelsie/__main__.py:51]: btw: you can just write `.extend(x)` not `.extend(list(x))`

**Comment by Kobzol** [comment]
Fixed the logic. Also sorted the images, to keep their order deterministic in the output.

**Review by spirali — APPROVED** [review]

**Pull request merged**